release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -775,6 +775,11 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-max-log-length-sidecar"]
+==== Increased maximum log length while using a sidecar
+Previously, the maximum length of the syslog message in the Ingress Controller was 1024 bytes. Now, the maximum value can be increased.
+// For more information, see xref
+:../networking/ingress-operator.adoc#nw-configure-ingress-access-logging_configuring-ingress[Allowing IngressController to modify the HAProxy log length when using a sidecar].
```

Comment:
There's a problem with the syntax here—the xref isn't rendering properly:

![Screenshot from 2023-10-26 23-10-50](https://github.com/openshift/openshift-docs/assets/2042465/7de5a79a-e500-4456-af12-f76c48050294)